client/src/js/leafletComponents/resultItem.js

```diff
@@ -13,9 +13,11 @@ var localization = require('../localization');
 var resultItem = {
   create: function (options) {
     this._options = options;
+    var colCount = options.isSearchResult ? 'col-xs-9 result'  : 'col-xs-6 c-result-item';
```

Comment:
I changed 'result' to be something more specific. I left 'c-result-item' 
because it was the original class from below that I moved up here so I 
didn't think you'd want me to change it.
On 06/15/2015 06:14 PM, lifehackett wrote:
>
> In client/src/js/leafletComponents/resultItem.js 
> <https://github.com/MobilityLabs/medellin-heritage/pull/21#discussion_r32477578>:
>
> > @@ -13,9 +13,11 @@ var localization = require('../localization');
> >  var resultItem = {
> >    create: function (options) {
> >      this._options = options;
> > +    var colCount = options.isSearchResult ? 'col-xs-9 result'  : 'col-xs-6 c-result-item';
>
> There isn't anyway you would know this, but staying with the 
> convention the css class names for this would be something like
> |.c-result-item--selected| & |.c-result-item--search|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MobilityLabs/medellin-heritage/pull/21/files#r32477578>.
>